File: 프로그래머스/1/42748. K번째수/K번째수.py
import copy
def solution(array, commands):
    answer = []
    
    for i, j, k in commands:
        new_array = copy.deepcopy(array)
        new_array = new_array[i-1:j]
        new_array.sort()
        answer.append(new_array[k-1])
    
    return answer

File: 프로그래머스/1/42748. K번째수/K번째수.java
import java.util.*;
class Solution {
    public int[] solution(int[] array, int[][] commands) {
        List<Integer> answer = new ArrayList<>();
        
        for(int[] command: commands) {
            List<Integer> a = new ArrayList<>();
            for(int i=command[0]-1; i<command[1]; i++){
                a.add(array[i]);
            }
            
            Collections.sort(a);
            answer.add(a.get(command[2]-1));
        }
        return answer.stream().mapToInt(i->i).toArray();
    }
}